src/main/java/com/google/sps/data/Event.java

```diff
@@ -0,0 +1,42 @@
+package com.google.sps.data;
+
+
+/**
+ * Event object for easy json packaing.
+ */
+public final class Event{
+
+    private String type;
+    private String details;
+    private int price;
```

Comment:
Should price be an int? Or a double?

Reply:
I agree, price should likely be a double or "float" if they exist in Java so that we can handle decimals.

Reply:
True. I'll change it to float since its range is more than enough.